core/shared/include/pragma/console/console.h

```diff
@@ -25,6 +25,13 @@ class DLLNETWORK DebugConsole {
 	std::ifstream _console_cin;
 	std::ofstream _console_cout;
 	std::ofstream _console_cerr;
+
```

Comment:
@SlawekNowy These appear to be unused, can they be removed?

Reply:
Yes.